### PR TITLE
Refactor to a python package

### PR DESCRIPTION
**Pull request by umesh-timalsina**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (justinGilmer) [inline LICENSE:22]: We need to copy over this section from mbuild's license too.

https://github.com/mosdef-hub/mbuild/blame/master/LICENSE.rst#L24-L62

**Review by justinGilmer — CHANGES_REQUESTED** [review]
Looks good so far, one small suggestion. Then I think we can merge. We will probably remove a bit of what the cookiecutter includes, but id rather do that in incremental PRs

File: LICENSE (hunk 1 above)
Comment (umesh-timalsina) [reply LICENSE:22]: Added

**Comment by umesh-timalsina** [comment]
Okay. This is ready to be merged.

**Comment by daico007** [comment]
I think it would be cool if we add a few examples of different way to create a `Box` object in `API.md`

**Comment by umesh-timalsina** [comment]
I have added simple example in `README.md`. I think we should include more examples in README.md itself, the reason being `API.md` is auto-generated via `lazydocs` with some minor edits.

**Pull request merged**